src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
+
+export const removeBook = () => ({
+  type: REMOVE_BOOK,
+});
+
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case addBook.fulfilled.type:
+    case removeBook.fulfilled.type:
+      return action.payload;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20state%20changes%20for%20the%20actions%20that%20you%20created.), Please make the function use ES6 filter() method to create a new array, which will not contain the book you want to remove from the store.
[Hint] It will be something like that:
```javascript
return state.filter(book => book.id !== action.id)
```

Reply:
done